webapp/src/js/components/Plot/Table/Actions.js

```diff
@@ -87,7 +90,14 @@ let TablePlotActions = React.createClass({
       label: table.capNamePlural
     }));
 
-
+    this.definedQuery = query;
```

Comment:
I don't understand why this is stored on the component (this) as opposed to being a local variable?